$OpenBSD$

Handle the GHC TypeLits export change for the :<>: error-message operator.
GHC 9.14+ exports (:<>:) without the "type" prefix.

--- src/Simplex/Chat/Messages.hs.orig
+++ src/Simplex/Chat/Messages.hs
@@ -41,7 +41,11 @@
 import Data.Time.Clock (NominalDiffTime, UTCTime, diffUTCTime, nominalDay)
 import Data.Type.Equality
 import Data.Typeable (Typeable)
+#if __GLASGOW_HASKELL__ >= 914
+import GHC.TypeLits (ErrorMessage (ShowType, (:<>:)), TypeError)
+#else
 import GHC.TypeLits (ErrorMessage (ShowType, type (:<>:)), TypeError)
+#endif
 import qualified GHC.TypeLits as Type
 import Simplex.Chat.Markdown
 import Simplex.Chat.Messages.CIContent
